FrontEnd/components/Sidebar/Right/index.vue

```diff
@@ -50,21 +55,51 @@ const whatsHappeningItems = ref([
 const { whoTofollow,handleFollowUnfollow} = useUser();
 
 const whoTofollowItems = ref([]);
+const loading = ref(false)
+
+
+// const fetchData = async () => {
+//   try {
+//     const response = await whoTofollow();
+//     console.log("who to follow response: ", response);
+
+//     whoTofollowItems.value = response.map(item => ({
+//       ...item,
+//       buttonState: 'follow'
+//     }));
+//   } catch (error) {
+//     console.log(error);
+//   } finally {
+//     loading.value = false;
+//   }
+// };
 
-onBeforeMount(async () => {
+// fetchData();
+const { data: whoToFollow, error, status } = await useAsyncData('whoToFollow', async () => {
   try {
-    const response = await whoTofollow();
-    console.log("who to follow response: ", response);
+    return await whoTofollow();    
```

Comment:
`await` not required when using `return`

Reply:
Removed await while calling whoToFollow() as mentioned .please review the last commit for the change.